Week3/More Javascript/script.js

```diff
@@ -0,0 +1,148 @@
+//1
+
+function info(a, b, c) {
+    console.log(a + b + c)
+}
+info(1, 5, 34);
+
+//2
+
+function colorCar(z) {
+    print = `a ${z} car`
+    console.log(print);
+}
+colorCar("red");
+
+//3
+
+let obj = {
+    name: "Ahmet Husrev",
+    surname: "Semiz",
+    sex: "Male",
+};
+
+function objPrint(obj) {
+
+    console.log(obj);
+}
+objPrint(obj);
+
+//4
+
+function vehicleType(a, b) {
+    if (b === 1) {
+        console.log(`a ${a} car`) ;
+    } else if (b === 2) {
+        console.log(`a ${a} motorbike`) ;
+    }
+}
+vehicleType("blue", 2);
+
+//5 
+
+console.log(3 === 3 ? "yes" : "no");
+
+//6
+
+let motor = [0,"car", "motorbike"];
+
+function vehicle(color, value, index) {
+    if (index === 4) {
+        return `a ${color} new ${motor[value]}`
+    }else if ( index === 5){
+        return `a ${color} used ${motor[value]}`
+    }else{
+        return `invalid access`
+    }
+}
+
+console.log(vehicle("blue", 1, 5));
```

Comment:
What if you do `console.log(vehicle("blue", 0, 5));` here? What will it print?
Or what if you do `console.log(vehicle("blue", 1, 1));` here? What will it print?

And nice! You have better variable names here, so now it's clear what the variables are for (`color` `value` and `index`

Reply:
Create a function called vehicle, like before, but takes another parameter called age, so that vehicle("blue", 1, 5) prints 'a blue used car'      I just create for vehicle("blue", 1, 5)  other values is undefined or random.